src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -253,7 +253,7 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
                 return;
             }
 
-            if (node.IsKind(SyntaxKind.ListPattern) && node.Parent != null)
+            if (node.Parent != null && node.Kind() is SyntaxKind.ListPattern or SyntaxKind.CollectionExpression)
```

Comment:
generally following the correspondance that list patterns and collection expressions should be formatted similarly.